rules/sinks/third_parties/sdk/sequelize/javascript.yaml

```diff
@@ -0,0 +1,13 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.SDK.Sequelize
```

Comment:
Sequelize seems to be an ORM. can we move this to database section. Also the pattern added will not get much matches, need some impl similar to Django ORM for this. Please remove this rule